webcompat/views.py

```diff
@@ -274,7 +275,16 @@ def contributors():
 @app.route('/tools/cssfixme')
 def cssfixme():
     '''Route for CSS Fix me tool'''
-    return render_template('cssfixme.html')
+
+    url = request.args.get('url')
```

Comment:
You can add tests in https://github.com/webcompat/webcompat.com/blob/master/tests/test_urls.py